wolfcrypt/src/tfm.c

```diff
@@ -4780,6 +4780,12 @@ int mp_toradix (mp_int *a, char *str, int radix)
         ++digs;
     }
 
+    /* For hexadecimal output, add zero when number of digits is odd */
```

Comment:
Consider also making this change in integer.c `mp_toradix`. Are we confident this zero padding is acceptable in all cases? Perhaps consider a build-time way to enable or disable this.

Reply:
Fixed. Thanks and good point on the integer.c. At the moment I can't think of a case where the zero-padding would be an issue. For now, I've added OPENSSL_EXTRA around the changes to minimize any negative impact. Let me know if you could think of an edge case.

Reply:
Perhaps it would be safer to use a new define like `WC_DISABLE_RADIX_ZERO_PAD` around it. Let it be on by default, but provide a way to disable at build-time?

Reply:
Fixed. Thanks